support-frontend/app/controllers/SubscriptionsController.scala

```diff
@@ -69,13 +68,14 @@ class SubscriptionsController(
     } else
       Map.empty
 
-    val fulfilmentOptions = if (countryGroup == CountryGroup.RestOfTheWorld) RestOfWorld else Domestic
-    val guardianWeeklyPromos = List(ausElection)
+    val guardianWeeklyFulfilmentOptions = if (countryGroup == CountryGroup.RestOfTheWorld) RestOfWorld else Domestic
+    // The client currently only supports monthly
+    val guardianWeeklyBillingPeriod = postIntroductorySixForSixBillingPeriod
```

Comment:
Is this just for clarity? You could rename the import instead:
```scala
import com.gu.support.catalog.GuardianWeekly.{postIntroductorySixForSixBillingPeriod => guardianWeeklyBillingPeriod}
```

Reply:
It's mainly to tie the fixed billing period (for GW) to the new comment. The fact that it doesn't support annual pricing/offers on the /support page is important to note (I think)